### PR TITLE
Sto mend updates doc 3682

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request

Preview link: [Mend scanner reference](https://64fa0c3de0da2b2fc8af761a--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/mend-scanner-reference)

Related tickets:
- [DOC-3682 STO doc :: Mend :: Add pipeline example and clarify terminology](https://harness.atlassian.net/browse/DOC-3682)
- [DOC-2458 STO update mend runner to support container scans](https://harness.atlassian.net/browse/DOC-2458)
- [DOC-2436 STO doc :: Scanner Reference :: Need to add info about Mend](https://harness.atlassian.net/browse/DOC-2436)

### CODE REVIEW

**Comment by douglas-j-bothwell** [comment]
Implemented Dylan's edits:

CleanShot 2023-09-07 at 10 38 45

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md [hunk 1]
```diff
@@ -16,7 +16,28 @@ For Harness on-prem releases, go to [Harness Self-Managed Enterprise Edition Rel
...
+
+* Fixed an issue when using multiple HTTP Helm Chart repositories that could lead to an increase in CPU utilization on the delegate due to background connector validation tasks. This was caused by running Helm repository update during the validation tasks. (CDS-76433, ZD-48363)
+
+* Fixed a Nexus Artifact issue where a fetch would time out when a single group contained more than 50 artifacts. (CDS-73884, ZD-45052, ZD-47206)
+  
+<!-- NOTE RE FIXED ISSUE CDS-73884
+https://harness.atlassian.net/browse/CDS-73884?focusedCommentId=566535 we fixed this issue for first gen with the same ticket, hence it is showing up in 805 release notes. 
+-->
+
+### Version 80407
```
Comment (brian-f-harness) [inline docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:41]: Not sure if you intentionally brought in rls note changes with this commit

File: docs/security-testing-orchestration/sto-techref-category/mend-scanner-reference.md [hunk 2]
```diff
@@ -166,68 +158,65 @@ import StoSettingAuthType from './shared/step_palette/_sto-ref-ui-auth-type.md';
...
-<StoSettingToolProjectName />
+This field is required for Extraction scans. 
 
 
 #### Include 
 
-```mdx-code-block
-import StoSettingToolInclude from './shared/step_palette/_sto-ref-ui-tool-include.md';
-```
+If you're running an orchestrated scan on a code repository, you can use this setting to specify the  specific files to include in the scan. By default, a Mend scan includes all files in the code repository.
```
Comment (brian-f-harness) [inline docs/security-testing-orchestration/sto-techref-category/mend-scanner-reference.md:202]:
```suggestion
If you're running an orchestrated scan on a code repository, you can use this setting to specify the files to include in the scan. By default, a Mend scan includes all files in the code repository. 
```

**Review by brian-f-harness — COMMENTED** [review]
May need to remove release note updates from this PR?

**Comment by douglas-j-bothwell** [comment]
Last two commits reverted CD release notes (FirstGen and NextGen) to latest versions in main to undo the accidental inclusion of 805xx content.

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md (hunk 1 above)
Comment (douglas-j-bothwell) [reply docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md]: @brian-f-harness This was my mistake. I just updated this topic with the latest content in main to undo these changes.

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -34,18 +34,85 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
 <Kustomizedep />
 
-
-## Latest: Version 80406
+## Latest: Version 805xx
 
 ### New features and enhancements
```
[the remark this answers is not in the document]
Comment (douglas-j-bothwell) [reply release-notes/continuous-delivery.md]: @brian-f-harness This was my mistake. I just updated this topic with the latest content in main to undo these changes.

[the remark this answers is not in the document]
Comment (brian-f-harness) [reply release-notes/continuous-delivery.md]: @douglas-j-bothwell It looks like the CG file and a couple of images are still in the PR.

**Pull request merged**